core/src/main/scala/org/apache/spark/deploy/mesos/MesosClusterDispatcher.scala

```diff
@@ -50,7 +50,11 @@ private[mesos] class MesosClusterDispatcher(
   extends Logging {
 
   private val publicAddress = Option(conf.getenv("SPARK_PUBLIC_DNS")).getOrElse(args.host)
-  private val recoveryMode = conf.get("spark.mesos.deploy.recoveryMode", "NONE").toUpperCase()
+  private val recoveryMode = conf.getOption("spark.mesos.deploy.recoverMode").map { mode =>
+    logWarning("spark.mesos.deploy.recoverMode is deprecated. Please configure " +
```

Comment:
I see, should we still honor the setting as a deprecation cycle? I'm going to just remove the warning for now but add a TODO.